Framework/Core/include/Framework/AnalysisTask.h

```diff
@@ -31,6 +31,7 @@
 
 #include <arrow/compute/kernel.h>
 #include <arrow/table.h>
+#include <arrow/util/key_value_metadata.h>
```

Comment:
Can we avoid exposing key_value_metadata.h by default?

Reply:
I'm looking into it. Without this include compilation fails with undefined sizeof on a forward-declared KeyValueMetadata, but only with some workflows.